eventlog/models.py

```diff
@@ -0,0 +1,39 @@
+from django.db import models
+from uuid import uuid4
+from roster.models import Period, ClusiveUser, Roles
+from django.utils import timezone
+import caliper
+
+# A user session
+class Session(models.Model):
+    id = models.CharField(primary_key=True, default=uuid4, max_length=36)
+    user = models.ForeignKey(to=ClusiveUser, on_delete=models.CASCADE)
```

Comment:
We should confirm that the `on_delete` behaviour is what's desirable for all of the ForeignKey relationships, as there's been talk in the past of the issue of deleting a user but retaining anonymized data.

Reply:
Setting to PROTECT for now. That should prevent deletes of users, sessions, groups if they have event data attached to them.  We can figure out alternative methods later, but this should keep us safe for now.